geowebcache/pom.xml

```diff
@@ -521,6 +534,28 @@
           <artifactId>cobertura-maven-plugin</artifactId>
           <version>2.0</version>
         </plugin>
+        <plugin>
+          <artifactId>maven-failsafe-plugin</artifactId>
+          <version>3.3.1</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>integration-test</goal>
+                <goal>verify</goal>
+              </goals>
+            </execution>
+          </executions>
+          <inherited>true</inherited>
+          <configuration>
+            <systemPropertyVariables>
+              <java.util.logging.config.file>src/test/resources/logging.properties</java.util.logging.config.file>
+              <!-- prevent ForkedBooter focus steal on macOS -->
+              <java.awt.headless>true</java.awt.headless>
+            </systemPropertyVariables>
+            <argLine>${maven.test.jvmargs} -XX:+IgnoreUnrecognizedVMOptions --illegal-access=warn --add-exports=java.desktop/sun.awt.image=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED</argLine>
+            <skipTests>${skipITs}</skipTests>
```

Comment:
skipITs seems to be [the default way to have failsafe skip running the integration tests](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/skipping-tests.html).

The skipTests parameter is [apparently deprecated](https://maven.apache.org/surefire/maven-failsafe-plugin/integration-test-mojo.html).

Reply:
Meanwhile, `skipTests` is the standard way to skip running the tests but compiling them. There's a reason to set `<skipTests>${skipITs}</skipTests>` in the failsafe configuration. Adding the following comment:
```xml
<!-- allow skipping unit tests but run integration tests with -DskipTests -DskipITs=false, otherwise skipTests will also disable the failsafe run -->
```